doc/welcome.zh.tm

```diff
@@ -3,119 +3,104 @@
 <style|<tuple|tmdoc|chinese|old-spacing|old-dots|old-lengths|compact-list>>
 
 <\body>
-  <tmdoc-title|欢迎使用墨干>
```

Comment:
I prefer to display Chinese in `zh.tm`.

Reply:
Is there a convenient way to do this? I write in English a lot, and it seems unacceptable to turn on and off the setting constantly.

Reply:
Fixed it.